roles/openshift_health_checker/test/etcd_volume_test.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+
+from openshift_checks.etcd_volume import EtcdVolume, OpenShiftCheckException
+
+
+@pytest.mark.parametrize('ansible_mounts,extra_words', [
+    ([], ['none']),  # empty ansible_mounts
+    ([{'mount': '/mnt'}], ['/mnt']),  # missing relevant mount paths
+])
+def test_cannot_determine_available_disk(ansible_mounts, extra_words):
+    task_vars = dict(
+        ansible_mounts=ansible_mounts,
+    )
+    check = EtcdVolume(execute_module=fake_execute_module)
+
+    with pytest.raises(OpenShiftCheckException) as excinfo:
+        check.run(tmp=None, task_vars=task_vars)
+
+    for word in 'determine available disk'.split() + extra_words:
+        assert word in str(excinfo.value)
+
+
+@pytest.mark.parametrize('ansible_mounts', [
+    [{
+        'mount': '/',
+        'size_available': 40 * 10**9 + 1,
+        'size_total': 80 * 10**9 + 1
+    }],
+    [{
+        'mount': '/',
+        'size_available': 30 * 10**9,
+        'size_total': 30 * 10**9 + 1,
+    }],
+    [{
+        'mount': '/',
+        'size_available': 20 * 10**9 + 1,
+        'size_total': 40 * 10**9 + 1,
+    }],
+    [{
+        # not enough space on / ...
+        'mount': '/',
+        'size_available': 0,
+        'size_total': 0,
+    }, {
+        # not enough space on /var/lib ...
+        'mount': '/var/lib',
+        'size_available': 2 * 10**9 + 1,
+        'size_total': 21 * 10**9 + 1,
+    }, {
+        # ... but enough on /var/lib/etcd
+        'mount': '/var/lib/etcd',
+        'size_available': 36 * 10**9 + 1,
```

Comment:
Are the `+ 1`'s in the test cases here actually relevant?

I think in call cases the default of 90% of the total will be used as the limit? The plus ones seem to be just distraction, no?